src/styles/Base/theme.scss

```diff
@@ -0,0 +1,38 @@
+/* Resets */
+
+ul {
+
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+
+  text-decoration: none;
+  color: #fff;
```

Comment:
You know because the default background is white like pure white I suggest using another color for the links by default. Even though our team is dark. You could look into another white-ish color. Look at material design colors in google and find a white-ish color if you want to stick with white but I think you should change it to a darker color.